src/plugins/workspace/public/plugin.ts

```diff
@@ -75,11 +78,37 @@ export class WorkspacePlugin implements Plugin<{}, {}, WorkspacePluginSetupDeps>
     });
   }
 
-  public async setup(core: CoreSetup, { savedObjectsManagement }: WorkspacePluginSetupDeps) {
+  /**
+   * If workspace is enabled and user has entered workspace, hide advance settings and dataSource menu and disable
```

Comment:
Is this sentence incomplete?

Reply:
Perhaps this is what you mean?
```suggestion
   * If workspace is enabled and user has entered workspace, hide advance settings and dataSource menu by disabling the corresponding apps.
```